pkg/service/service.go

```diff
@@ -238,6 +305,13 @@ func (s *Service) InitMaps(ipv6, ipv4, sockMaps, restore bool) error {
 		}
 	}
 
+	if v2BackendMapExistsV4 || v2BackendMapExistsV6 {
+		log.Info("Backend map v2 exists. Migrating entries to backend map v1, may lose entries")
```

Comment:
IMHO "may lose entries" might scare users. The err msg for that case is enough.

Reply:
Done.